.github/workflows/backport-prs.yml

```diff
@@ -27,8 +27,9 @@ jobs:
         contains(github.event.comment.body, '/backport')
       )
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Create backport pull requests
         uses: korthout/backport-action@v1
         with:
           merge_commits: 'skip'
+          persist-credentials: false
```

Comment:
Was this mean to go with the `checkout` step?

Reply:
Also, why is this parameter needed?

Reply:
`persist-credentials` defaults to true, which means it doesn't delete the auth token from disk after cloning. Ensuring the token is deleted after this step reduces the attack surface of any malicious commits that might slip through review.

The [current guidance](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/) is to set `persist-credentials: false` unless we need to do additional remote git operations after this step.

Reply:
> Was this mean to go with the checkout step?

Oh yes it was. I will fix that

Reply:
Done